content/posts/bitcoin.md

```diff
@@ -38,26 +38,26 @@ beneficial exchange to drive progress. But the modern incarnation of military-in
 monopolistic, American consumerism is not particularly nurturing for a sustainable future. 
 Tesla is perhaps American capitalism's best foot forward.
 
-Two is the supply chain of modern technology.
+Another is the supply chain of modern technology.
 The mining industry, for metals like Lithium, Coltan, and more,
 is [riddled with environmental and human rights abuses][supply-chain].
 These are not so much measured in carbon emissions (though there are plenty)
 as in pollution and destruction of natural habitats and social relationships.
 And a car battery requires many orders of magnitude more Lithium than your Macbook. 
 Clean energy has a very dirty supply chain, and little incentive to improve it.
 
-Third is cars themselves. The complete centrality of cars in the
+A third is cars themselves. The complete centrality of cars in the
 organization of urban life is one of the great tragedies
 of our time. Sustainability is intimately bound up with how we organize ourselves
 on the surface of the planet. Cars, electric or otherwise, are not
 particularly conducive to the more human-scale and bio-diverse patterns that are 
-critical to sustainability. To the extent we have cars they should probably be electric. 
+critical to sustainability. To the extent that we have cars, they should probably be electric. 
 But more than anything, we need less cars. Denmark has the right idea. As does
 anyone who's ever gotten a street closed to cars, for any duration. Respect.
 
 Now I'm not trying to diss Tesla, per se. 
 Tesla is the beacon of a research and development process that is itself critical to
-sustainability. And if we want to make progress in batteries, which we should, cars are a great
+sustainability. <!-- I don't follow the reasoning of the next sentence: ---> And if we want to make progress in batteries, which we should, cars are a great
 way to go. But the fundamental problems we're facing today don't seem so much technical as they
```

Comment:
ah, just that they provide a good socioeconomic impetus for battery. car gas tanks are one of the most dominant energy stores in peoples experience so from a product-market perspective, cars are probably a good place battery innovation to get funded and delivered

Reply:
That makes sense. So, iiuc, the reasoning is something like: given the current way we motivate and incentivize research, increasing the demand for, and so the profits to be gained from, electronic cars, may be the best bet to getting research directed to battery technology? I see how this could parallel the argument about bitcoin demand driving green tech innovation.